source/layout.erb

```diff
@@ -62,9 +62,23 @@
 
     <div id="content-wrapper">
       <div id="sidebar-wrapper">
-        <select class="version-select" style="width: 100%"></select>
-        <div id="sidebar">
+        <div id="versions">
+          <noscript>
+            <ul>
+              <% data.versions.each do |version| %>
```

Comment:
How would previously published revisions of the site know about the new set of versions?